R/update_ard.R

```diff
@@ -81,7 +82,7 @@ update_ard_fmt_fn <- function(x, variables = everything(), stat_names, fmt_fn, f
   }
 
   # update ARD with new fmt_fn -------------------------------------------------
-  x$fmt_fn[idx1 & idx2] <- list(alias_as_fmt_fn(fmt_fn))
+  x$fmt_fn[idx1 & idx2] <- list(fmt_fn)
```

Comment:
why not anymore?

Reply:
Ahh, good catch! In the first version i wrote I needed to make this change, but now we don't. I'll revert this, thank you!

Reply:
Updated!